projects/sdk/src/utils/TestUtils/BlockchainUtils.ts

```diff
@@ -80,7 +80,7 @@ export class BlockchainUtils {
     await this.sdk.provider.send("anvil_reset", [
       {
         forking: {
-          jsonRpcUrl: "https://eth-mainnet.g.alchemy.com/v2/f6piiDvMBMGRYvCOwLJFMD7cUjIvI1TP"
+          jsonRpcUrl: "PROVIDER_RPC_URL"
```

Comment:
This ENV var doesn't exist anywhere else; this change would break things

Reply:
Would it be ok if i added a .env file on the TestUtils folder and got the value from process.env? So that everyone could use their own rpc url.

Reply:
Rephrase: Should i get the rpc url through process.env? So that everyone can create their own .env file and put any api keys or urls here.

Reply:
Changed. Should work now with existing .env-sample file.